templates/try-matrix.html

```diff
@@ -23,7 +43,7 @@ <h3>Choose a client</h3>
             <img src="/assets/try-matrix/client.svg" alt="Artist view of an abstract matrix client." />
         </div>
     </div>
-    <div class="step-background coloured">
+    <div class="step-background">
```

Comment:
Not a blocker for the PR, but it'd be nice to use `:nth-child()` instead of manually adding/removing the class

https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#keyword_values